airbyte-cdk/python/unit_tests/sources/test_abstract_source.py

```diff
@@ -78,7 +83,7 @@ class MockStreamOverridesStateMethod(Stream, IncrementalMixin):
     primary_key = None
     cursor_field = "updated_at"
     _cursor_value = ""
-    start_date = "1984-12-12"
+    start_date = "19-12-12"
```

Comment:
this date looks weird. Why is the year changed to `19`?

Reply:
Definitely not intentionally.  Will revert this.